TX-Ultimate-Easy-ESPHome_advanced.yaml

```
@@ -0,0 +1,38 @@
+#####                              TX Ultimate Easy for ESPHome                                #####
+#####                  Repository: https://github.com/edwardtfn/TX-Ultimate-Easy               #####
+####################################################################################################
+##### Author: edwardtfn - https://github.com/edwardtfn - https://buymeacoffee.com/edwardfirmo  #####
+####################################################################################################
+##### WARNING: This is an advanced configuration template. Use with caution as improper        #####
+##### configuration may lead to reduced functionality or system instability.                   #####
+####################################################################################################
+---
+substitutions:
+  name: tx-ultimate-easy-adv
+  friendly_name: TX Ultimate Easy Advanced  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)